### ISSUE TITLE
Add ability to visualize train signals

### ISSUE THREAD

**Issue by Happyrobot33** [L0laapk3/FactorioMaps#114]
it would be nice if there was a way to turn on train signal rendering like exists on the ingame map. I use a lot of train signal displays in my world and these are just unreadable on the web map due to the way they function

**Comment by L0laapk3** [comment]
Are you talking about train block sections?

**Comment by Happyrobot33** [comment]
kind of, I use these kind of displays a lot where they use the train signals as the pixels
![image](https://user-images.githubusercontent.com/32679887/218237476-5f847c29-ea59-4e3b-975f-de5f90237b7f.png)
![image](https://user-images.githubusercontent.com/32679887/218237481-cc5a570d-c4fb-4c54-ba77-3d5369dfc415.png)

**Comment by L0laapk3** [comment]
Does this not already render with altmode? (Which is on by default)

**Comment by Happyrobot33** [comment]
no, since these dont actually have a alt mode in the base game. they only show up like this on the map

**Comment by Happyrobot33** [comment]
if you want more of an example of what I'm talking about, got to [my hosted map](factorio.matthewherber.com)

**Comment by L0laapk3** [comment]
I see.

Factoriomaps has been on maintenance mode for quite a while for me, so its
unlikely that I'll add this.
However if you want to give it a go yourself, I'm more than happy to advise
on how to add this and review/accept pull requests.
